### PR TITLE
TypeScript @ts-ignore: estimateFeeValue

### PR DESCRIPTION
**Pull request by NotEternal**
## Checklist

- [x] I have read the [CONTRIBUTING](https://github.com/swaponline/swap.react/wiki/CONTRIBUTING) guide
- [x] Good naming, keep simple
- [ ] Tested desktop/mobile
- [ ] Tested bright/dark
- [ ] Tested en/ru
- [ ] Affects money; I checked the functionality once again
- [x] I checked the PR once again

### Original issue
#3562

### CODE REVIEW

File: src/front/shared/helpers/btc.ts [hunk 1]
```diff
@@ -149,8 +149,18 @@ const calculateTxSize = async ({ speed, unspents, address, txOut = 2, method = '
   return txSize
 }
 
-//@ts-ignore
-const estimateFeeValue = async ({ feeRate, inSatoshis, speed, address, txSize, fixed, method }: object = {}) => {
+type EstimateFeeValueOptions = {
+  method?: string
+  speed: string
```
Comment (iower) [inline src/front/shared/helpers/btc.ts:154]:
```suggestion
  speed: 'fast' | 'normal' | 'slow'
```

File: src/front/shared/helpers/ghost.ts [hunk 2]
```diff
@@ -160,8 +160,18 @@ const calculateTxSize = async ({ speed, unspents, address, txOut = 2, method = '
   return txSize
 }
 
-//@ts-ignore
-const estimateFeeValue = async ({ feeRate, inSatoshis, speed, address, txSize, fixed, method } = {}) => {
+type EstimateFeeValueOptions = {
+  method?: string
+  speed: string
```
Comment (iower) [inline src/front/shared/helpers/ghost.ts:165]:
```suggestion
  speed: 'fast' | 'normal' | 'slow'
```

File: src/front/shared/helpers/next.ts [hunk 3]
```diff
@@ -159,8 +159,18 @@ const calculateTxSize = async ({ speed, unspents, address, txOut = 2, method = '
   return txSize
 }
 
-//@ts-ignore
-const estimateFeeValue = async ({ feeRate, inSatoshis, speed, address, txSize, fixed, method } = {}) => {
+type EstimateFeeValueOptions = {
+  method?: string
+  speed: string
```
Comment (iower) [inline src/front/shared/helpers/next.ts:164]:
```suggestion
  speed: 'fast' | 'normal' | 'slow'
```

**Review by iower — CHANGES_REQUESTED** [review]

**Comment by iower** [comment]
# Requesting 2 approvals for this PR

**Review by shendel — APPROVED** [review]

**Review by iower — APPROVED** [review]

**Pull request merged**